### ISSUE TITLE
When the Api Controller contains an Action with HttpPost but contains no binding model, bad JS is generated

### ISSUE THREAD

**Issue by alex-klock** [Alchemy4Tridion/Alchemy4Tridion#87]
With a scenario where you have a POST action that contains no model like:

```
[HttpPost]
[Route("somewhere")]
public bool MyControllerAction()
{

}
```

It will produce in the js:

```
var promise = qwest.post('/Alchemy/Plugins/My_Plugin_Name/api/MyService/somewhere/'.format(), );
```

The comma at the end w/o an expression after it causes the JavaScript to throw an error.

**Comment by alex-klock** [comment]
Fixed and ready for 0.6.1 release. Tested scenarios with no parameters, with route parameters but no model, with model only, and with route parameters and model.  

The model still must be the last parameter in the method declaration if a model is provided.

**Comment by robhimself82** [comment]
Did you happen to test this fix with minification disabled in the configuration of Tridion? Even with 0.6.1 installed, it's still generating the extra comma. As long as I go into the file system after installation of the plugin and make the change in the Proxy it works as intended, but otherwise no dice.
